### PR TITLE
Update header from marketing to include Connect banner

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: src/css/globals/vars.css [hunk 1]
```diff
@@ -64,7 +64,7 @@
   --z-gdpr: 20;
   --z-modal-header: 16;
   --z-modal: 15;
-  --z-nav-mobile: 14;
+  --z-nav-mobile: 1000;
```
Comment (sanderson-sfdc) [inline src/css/globals/vars.css:67]: Why did this have to bump up to override the other ones?  Did you test if search or whatever is a modal still works with this change?

File: src/css/specific/header.css [hunk 2]
```diff
@@ -1,13 +1,33 @@
 /* overrides for marketing header */
-.ms-com-content-header {
+body.flex .ms-com-content.ms-com-content-header {
   border-bottom: 1px solid var(--aluminum-3);
   box-shadow: 0 3px 15px rgba(0,0,0,.025);
+  z-index: 1000;
```
Comment (sanderson-sfdc) [inline src/css/specific/header.css:5]: Rather than hard-coding this, can you use a variable?

File: src/css/specific/navigation.css [hunk 3]
```diff
@@ -22,7 +22,7 @@
     position: sticky;
     top: 0;
     transform: translateX(0);
-    z-index: var(--z-nav);
+    z-index: 0;
```
Comment (sanderson-sfdc) [inline src/css/specific/navigation.css:25]: As above, can you use a variable here?

File: src/css/globals/vars.css (hunk 1 above)
Comment (fermujica) [reply src/css/globals/vars.css:67]: As of today, we can't test the search bar unless we build this on stgx.

Comment (sanderson-sfdc) [reply src/css/globals/vars.css:67]: Ah, right.  Do what you think is best, then.

**Comment by fermujica** [comment]
Turned the z-index values into variables. 

The values were suggested by marketing team while we troubleshooted some overlapping issues with these changes.

**Pull request merged**